scheduler/src/cook/components.clj

```diff
@@ -37,7 +37,8 @@
             [ring.middleware.params :refer (wrap-params)]
             [ring.middleware.stacktrace :refer (wrap-stacktrace)]
             [ring.util.mime-type]
-            [ring.util.response :refer (response)])
+            [ring.util.response :refer (response)]
+            [ratelimit])
```

Comment:
Nitpick: I would prefer `rate-limit`.

Reply:
There's not a bit of cook-dependent code in that namespace; e.g., we could take the code and move it to waiter, wholesale.